client/gutenberg/extensions/related-posts/constants.js

```diff
@@ -5,20 +5,26 @@ export const MAX_POSTS_TO_SHOW = 3;
 export const DEFAULT_POSTS = [
 	{
 		title: 'Big iPhone/iPad Update Now Available',
-		image: 'https://wordpress.com/calypso/images/related-posts/cat-blog.png',
-		date: '2018-08-03',
+		img: {
+			src: 'https://wordpress.com/calypso/images/related-posts/cat-blog.png',
+		},
+		date: 'August 3, 2018',
 		context: 'In "Mobile"',
 	},
 	{
 		title: 'The WordPress for Android App Gets a Big Facelift',
-		image: 'https://wordpress.com/calypso/images/related-posts/devices.jpg',
-		date: '2018-08-02',
+		img: {
+			src: 'https://wordpress.com/calypso/images/related-posts/devices.jpg',
+		},
+		date: 'August 2, 2018',
 		context: 'In "Mobile"',
 	},
 	{
 		title: 'Upgrade Focus: VideoPress For Weddings',
-		image: 'https://wordpress.com/calypso/images/related-posts/mobile-wedding.jpg',
-		date: '2018-08-05',
+		img: {
+			src: 'https://wordpress.com/calypso/images/related-posts/mobile-wedding.jpg',
+		},
+		date: 'August 5, 2018',
```

Comment:
It looks like there are a lot of strings here that can wind up in UI. Should these things be localized?

Reply:
If we want to land this and come back later, let's get an open issue to track this.